spec/factories.rb

```diff
@@ -291,6 +291,12 @@
       selected true
       feasibility "feasible"
       valuation_finished true
+
+    end
+
+    trait :winner do
+      selected
```

Comment:
This got me a bit confused, but `selected` is both an attribute of budget investment, and a trait of its factory :) So yes it works!